scripts/docker-lotus-entrypoint.sh

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env bash
 
-if [ ! -z $DOCKER_LOTUS_IMPORT_SNAPSHOT ]; then
+if [ ! -z "$DOCKER_LOTUS_IMPORT_SNAPSHOT" ]; then
 	GATE="$LOTUS_PATH"/date_initialized
 	# Don't init if already initialized.
 	if [ ! -f "$GATE" ]; then
 		echo importing minimal snapshot
-		/usr/local/bin/lotus daemon --import-snapshot "$DOCKER_LOTUS_IMPORT_SNAPSHOT" --halt-after-import
+		printf 'yes\n' | /usr/local/bin/lotus daemon --import-snapshot "$DOCKER_LOTUS_IMPORT_SNAPSHOT" --halt-after-import
```

Comment:
This actually seems like a bug. We should only be prompting the user if there's actually a chain to remove.

Reply:
https://github.com/filecoin-project/lotus/pull/11582

Reply:
Fixed on master, this change shouldn't be necessary anymore. Although it may help to pass `--remove-existing-chain=false` (which will cause an immediate failure if there's an existing change, which there shouldn't be given that the node isn't initialized).

Reply:
Rebased and added the flag. Although, I'm not sure what happened to the unreleased changelog.